src/main/resources/application.properties

```diff
@@ -10,7 +10,7 @@ spring.jpa.database=mysql
 
 spring.datasource.url=jdbc:mysql://localhost:3306/STUDY_DB?useSSL=false&characterEncoding=UTF-8&serverTimezone=Asia/Seoul&allowPublicKeyRetrieval=true&useSSL=false
 spring.datasource.username=root
-spring.datasource.password=fndlqhtm4132
+spring.datasource.password=root123123
```

Comment:
I think... this is your password.
right??
So we should fix username and password.
```
spring.datasource.username=root
spring.datasource.password=password
```

Reply:
right. It is my password. 
As if it was uploaded because of git ignore file not working.